### DISCUSSION TITLE
Welcome to Twinkle Discussions!

### DISCUSSION THREAD

**Discussion by Dorumin** [Dorumin/Twinkle#45] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Dorumin** [comment]
owo

**Reply by TheNozomi** [reply]
uwu

**Comment by lavgup** [comment]
hot

**Comment by owlnai** [comment]
Hey twinklers! I'm the guy who broke Twinkle with a shitpost PR that got actually merged and then fixed

**Comment by LunarTwilight** [comment]
what have you brought upon this land

**Comment by puxlit** [comment]
![:AoiNut:](https://cdn.discordapp.com/emojis/718325238089121893.png?size=64) ![:developers:](https://cdn.discordapp.com/emojis/768835610999783426.gif?size=64)
